src/main/resources/template/select.vm

```diff
@@ -1,29 +1,31 @@
 #* @vtlvariable name="action" type="com.atlassian.shele.shelePlugin.webwork.SelectAction" *#
 
-#set($ft = "Jira Loads ")
-#set($sc="Jira класс?")
-#set($title="Okay bro!")
+
+
 #set($context=$action.servletContext)
 <html>
 <head>
   <meta name="decorator" content="atl.admin"/>
   <meta name="admin.active.section" content="admin_plugins_menu/my-section"/>
   <meta name="admin.active.tab" content="my-next-item"/>
   <script src="https://connect-cdn.atl-paas.net/all.js"></script>
-  <title>$title</title>
+  <title>Okay bro!</title>
 </head>
 <body>
-<form class="aui" action="http://localhost:2990/jira/rest/rest-issue/1.0/api/issue" method="post" id="form-1">
+<form class="aui" action=$context.getAttribute("url")  method="post" id="form-1">
```

Comment:
I do not mean to replace the whole URL, it should look like below:
`$baseURL/rest/rest-issue/1.0/api/issue`